components/dom_crawler.rst

```diff
@@ -396,10 +396,19 @@ given text. This method is especially useful because you can use it to return
 a :class:`Symfony\\Component\\DomCrawler\\Form` object that represents the
 form that the button lives in::
 
-    $form = $crawler->selectButton('validate')->form();
+    // button example: <button id="my-super-button" type="submit">My super button</button>
+    
+    // you can get button my it's name
```

Comment:
So, it matches by button text? What if it's an input type="submit" - does it use the name attribute then?